sdk/search/Microsoft.Azure.Search/tests/Tests/IndexerTests.cs

```diff
@@ -409,6 +431,7 @@ public void CanCreateBlobIndexerWithConfigurationParameters()
         }
 
         [Fact]
+		[LiveTest]
```

Comment:
I will stop commenting on the indention in each file now. Please review and update all the test files you added the LiveTest attribute to and adjust the indention accordingly.

Reply:
I have those all resolved now.